Motorcycle.js

```diff
@@ -1,6 +1,23 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(seats, running) {
+	Car.call(this, seats, running);
+	this.seats = seats;
```

Comment:
To override when a user puts more than 2 seats in their motorcycle, we can add a conditional statement:
```
if (this.seats > 2) {
		this.seats = 2;
	}
```